web/src/constants/sign-up-errors.ts

```diff
@@ -14,8 +14,28 @@
  * limitations under the License.
  */
 
-export const NAME_EMPTY = 'Name cannot be empty.';
-export const EMAIL_EMPTY = 'Email cannot be empty.';
-export const PASSWORD_EMPTY = 'Password cannot be empty.';
-export const PASSWORDS_DO_NOT_MATCH = 'Passwords do not match.';
-export const VPA_EMPTY = 'VPA cannot be empty.';
+const NAME_EMPTY = 'Name cannot be empty.';
+const EMAIL_EMPTY = 'Email cannot be empty.';
+const PASSWORD_EMPTY = 'Password cannot be empty.';
+const VPA_EMPTY = 'VPA cannot be empty.';
+
+const PASSWORDS_DO_NOT_MATCH = 'Passwords do not match.';
+
+const INVALID_EMAIL = 'Please input a valid email address.';
+const WEAK_PASSWORD = 'Password should be at least 6 characters.';
```

Comment:
Why this error? Is there a error msg sent from firebase?

Reply:
Yeah firebase has these errors.